docs/governance/community-groups.md

```diff
@@ -22,6 +22,46 @@ Non-technical issues or high-level design decisions may be escalated to the [SC]
 
 Technical issues should be resolved by the relevant community group. Failure to resolve technical issues may be escalated to the [SC] for guidance on finding a resolution.
 
+## Working Groups
+
+The CCC project is split into several separate, but related, working groups. Each working group focuses on a specific sub-area of the CCC problem space.
+
+### Working Groups Structure
+
+Working groups are structured with distinct roles to ensure effective operation and governance
```

Comment:
I assume these roles are controlled by some IAM mechanism - is that correct? People can have none, one, or more of these roles ascribed to them at any given time? If so, how does one earn the right/honour/privilege to be granted a role? Is that something that's covered elsewhere in this doc? Or somewhere else entirely?

Reply:
I was thinking these would be roles assinged to people based on discussions. I don't think it would be controlled by anything more than having assigning a role to a person and updating the charter. If there was a more formal way to control this I would be open for it, but I am not sure if the overhead would be worth it.

In terms of how people are assigned a role that isn't covered anywhere. I will look to add something. My initial thinking is:

Working group lead - a volunteer from the working group
Maintainers - these would be people from the list of the project maintainers that currently exists. Each maintainer can then volunteer themselves to be part of a working group.
PR Reviewers - the critera for what makes someone a suitable PR reviewer would be determined on a group by group basis depending how many people are in the working group.
Group members - anyone who wants to participate and contribute to a working group is welcome to.

Does that sounds reasonable for a starting point?

Reply:
Changes pushed to address this.